assets/xml/objects/object_hakugin_demo.xml

```diff
@@ -1,8 +1,8 @@
 ﻿<Root>
     <File Name="object_hakugin_demo" Segment="6">
```

Comment:
Can you put a comment above the `<File>` node describing what's in this object? There are plenty of examples in the codebase to copy if you want to see how it's done, or you can ask in Discord.